docs/ruleset_functions/get_hvac_sys_and_assoc_zones_largest_exhaust_source.md

```diff
@@ -0,0 +1,41 @@
+# get_hvac_sys_and_assoc_zones_largest_exhaust_source  
+
+**Description:** Returns a list with the sum of the hvac fan system exhaust fan cfm values, the maximum zone level exhaust source across the zones associated with the HVAC system, the number of exhaust fans associated with the hvac fan system, and the maximum cfm of all of the exhaust fans associated with the hvac system fan system [hvac_sys_exhaust_cfm_sum, maximum_zone_exhaust, num_hvac_exhaust_fans, maximum_hvac_exhaust]. This is for evaluating G3.1.2.10 exception 6 which states "Where the largest exhaust source is less than 75% of the design outdoor airflow. This exception shall only be used if exhaust air energy recovery is not used in the proposed design. "
+
+**Inputs:**  
+- **U,P,or B-RMI**: The RMI in which the largest exhaust source CFM is to be returned.  
+- **hvac_sys**: The hvac system object for which the largest exhaust source is being determined.
+
+**Returns:**  
+- **get_hvac_sys_and_assoc_zones_largest_exhaust_source**: Returns a list with the sum of the hvac fan system exhaust fan cfm values, the maximum zone level exhaust source across the zones associated with the HVAC system, the number of exhaust fans associated with the hvac fan system, and the maximum cfm of all of the exhaust fans associated with the hvac system fan system [hvac_sys_exhaust_cfm_sum, maximum_zone_exhaust, num_hvac_exhaust_fans, maximum_hvac_exhaust]. This is for evaluating G3.1.2.10 exception 6 which states "Where the largest exhaust source is less than 75% of the design outdoor airflow. This exception shall only be used if exhaust air energy recovery is not used in the proposed design. "  
+ 
+**Function Call:**   
+1. get_dict_of_zones_and_terminal_units_served_by_hvac_sys()   
+
+
+## Logic:  
+- Create dictionary of hvac systems, zones, and terminal units: `dict_of_zones_and_terminal_units_served_by_hvac_sys = get_dict_of_zones_and_terminal_units_served_by_hvac_sys(B_RMI)`  
+- Get the list of zones associated with the hvac system: `zone_list = dict_of_zones_and_terminal_units_served_by_hvac_sys[hvac_sys.id]["ZONE_LIST"]`  
+- Set largest zone exhaust source cfm equal to zero: `maximum_zone_exhaust = 0`  
+- Set largest hvac exhaust cfm equal to zero: `maximum_hvac_exhaust = 0`  
+- Create a fan system object: `fan_system_p = sys.fan_system`   
```

Comment:
Why is there "p" suffix here and in other places? Shouldn't this apply to the baseline?

Reply:
This is a  mistake. Fixed.